src/Service/DistanceCalculator.php

```diff
@@ -28,16 +43,21 @@ public function calculateDistance(HasAddress $start, HasAddress $end): ?float
         $coordinatesStart=$resStart['features'][0]['geometry']['coordinates'];
         $coordinatesEnd=$resEnd['features'][0]['geometry']['coordinates'];
 
+        return self::calculateDistance($coordinatesStart, $coordinatesEnd);
```

Comment:
non tu n'es pas en static, c'est bien $this->calculateDistance() que tu dois utiliser ici